pkgs/by-name/py/pyflyby/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pyflyby";
+  version = "1.9.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "deshaw";
+    repo = "pyflyby";
+    rev = version;
+    hash = "sha256-QkoFr9tFtZ+ZEWlxe9csrzoYFl9/V2l4hKYfUWsXUdc=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
```

```suggestion
  build-system = with python3.pkgs; [
```
Not sure if `wheel` would come into this, but feel free to look around